templates/AdminLoginScreen.jsx

```diff
@@ -1,13 +1,79 @@
 // import axios from 'axios';
 // import { useCallback } from 'react';
 
+import { Field, Form, Formik } from 'formik';
+import * as Yup from 'yup';
+import AppInput from './Admin/Layout/Component/Form/AppInput';
+import { useState } from 'react';
+
 export default function AdminLoginScreen() {
+  const [validationSchema] = useState(
+    Yup.object().shape({
+      email: Yup.string()
+        .required('Introduzca un email')
+        .email('El email no es valido'),
+      password: Yup.string().required('Introduzca su contraseña'),
+    })
+  );
+
   // const handleTestUrl = useCallback(async () => {
   //   axios
   //     .get('http://localhost/public/index.php/api/test')
   //     .then((r) => console.log(r.data))
   //     .catch((err) => console.log(err));
   // });
 
-  return <>AdminLoginScreen </>;
-}
\ No newline at end of file
+  return (
+    <>
+      <div className="container">
+        <div className="row justify-content-center">
+          <div className="col-md-6">
+            <h2 className="text-center mt-5">Iniciar Sesión</h2>
+            <div className="card my-3 shadow">
+              <div className="card-body">
+                <Formik
+                  initialValues={{
+                    email: null,
+                    password: null,
+                  }}
+                  validationSchema={validationSchema}
+                  validateOnChange={false}
+                  validateOnBlur={false}
+                  enableReinitialize
```

Comment:
No uséis el enableReinitialize, a la larga puede dar problemas, y no aporta nada aquí